frontend/src/entities/batch-translate/create/batch-translate-create.tsx

```diff
@@ -76,6 +77,13 @@ export function BatchTranslateCreate () {
     scrollToPageHeader();
     DocTitle('Create translation job');
 
+    // the dependencies needed for language detection aren't available in us-iso-east-1
+    const supportsAutoLanguageDetection = window.env.AWS_REGION !== 'us-iso-east-1';
```

Comment:
If Translate is ever built into us-isob-east-1 or any other region where auto isn't supported it would be nice if we made this more extensible. 

Maybe we have a list defined here or in a constants/util class which defines all regions auto isn't supported in, and then we check if the user's region is in the list of unsupported regions:

```
const supportsAutoLanguageDetection = !autoSupportedRegions.includes(window.env.AWS_REGION);
```

Reply:
I agree that this should be a list of unavailable regions. I think the inclusive list would be much larger